### PR TITLE
Add -wegswitchgpu option to disable igpu if egpu is detected

### PR DESCRIPTION
**Pull request by dirkenstein**
Useful if MacBook/MBP igpu does not support metal and egpu does.

### CODE REVIEW

File: Lilu/Headers/kern_devinfo.hpp [hunk 1]
```diff
@@ -160,6 +160,12 @@ class DeviceInfo {
 	 */
 	static constexpr const char *RequestedInternalSwitchOffArg {"-wegnoigpu"};
 
+	/**
+	 *  The boot-arg to force-disable any internal GPU if external GPU found.
+	 *  For user configuration only! Use requestedExternalSwitchOff!
```
Comment (vit9696) [inline Lilu/Headers/kern_devinfo.hpp:165]: The comment mentions the wrong var.

**Comment by vit9696** [comment]
Why is -wegnoigpu not enoug?

**Comment by dirkenstein** [comment]
This way, you don't need a different boot option/selection for the MBP if
the eGPU is not plugged in.
With -wegnoigpu the startup hangs without any eGPU being present.
At least you can get to an unaccelerated internal screen this way.

**Comment by vit9696** [comment]
Right, this option still need feature parity with device properties. There should be a device property injected into the IGPU that does the same thing. Other than that, I guess it makes good sense.

**Comment by dirkenstein** [comment]
OK I will add that and correct the comment.

**Pull request merged**